tests/Unit/Utilities/Protocols/Test_StaticReturnApplyable.cpp

```diff
@@ -53,4 +69,7 @@ SPECTRE_TEST_CASE("Unit.Utilities.Protocols.StaticReturnApplyable",
       T1{}, T2{}, T3{}, T4{});
   db::mutate_apply<Applyable>(make_not_null(&box));
   CHECK(true);
```

Comment:
Only need one `CHECK(true)`.  It's just there to avoid a catch error when there are no checks.